contrib/download-frozen-image-v2.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```

Comment:
@tianon this is an absolute nit but, on `NixOS` this doesn't work (because `/bin/bash` doesn't exist). I took the habit to use `/usr/bin/env bash` in the shebang.